ydb/library/yql/core/common_opt/yql_co_pgselect.cpp

```diff
@@ -1519,6 +1530,63 @@ std::tuple<TVector<ui32>, TExprNode::TListType> BuildJoinGroups(TPositionHandle
                 current = cartesian;
                 continue;
             }
+            if (join->ChildrenSize() > 1 && join->Child(1)->Content() == "using") {
```

Comment:
please check that join->Child(1) is an Atom and add YQL_ENSURE for its content

join->ChildrenSize() > 1 looks like YQL_ENSURE too
graph should be valid at this moment